developers/README.md

```diff
@@ -2,75 +2,65 @@
 
 ## Table of Contents
 
-* [Development Environment](#development-environment)
-  * [Virtual Machines](#virtual-machines)
 * [Contributing](#contributing)
-* [IRC](#irc)
+* [Discussion](#discussion)
+* [Local Development Environment](#local-development-environment)
+  * [Local Testing](#local-testing)
+* [Ansible Environments](#ansible-environmnets)
+  * [Production](#production)
+  * [Multinode Nodepool](#multinode-nodepool)
+  * [Docker](#docker)
+  * [Vagrant](#vagrant)
 
-## Development Environment
+## Contributing
 
-Before hacking on BonnyCI, we'll need to set up a development environment on our local machine.
+Please see the [contributing documentation](contributing) for more information.
 
-Supported development platforms:
+A brief note on repository naming...
+> The puns get made, or the plank gets walked.
+>
+> ~Jesse Keating
 
-* [Ubuntu](dev-environment/ubuntu.md)
-* [Apple's macOS](dev-environment/macOS.md)
+## Discussion
 
-### Virtual Machines
+For discussions about BonnyCI, join us on [freenode](https://freenode.net) #BonnyCi!
 
-You can test changes to BonnyCI locally by exercising some tools defined in [hoist](www.github.com/BonnyCI/hoist). Hoist is a set of ansible playbooks that automate the deployment of a BonnyCI environment. Be sure that you have completed the `Virtualization Tools` section of your OS's [Development Environment](#development-environment) page before proceeding.
+## Ansible Environments
 
-Clone hoist and navigate to its base directory:
+There are four different ansible environments that hoist supports:
 
-```shell
-$ git clone git@github.com:BonnyCI/hoist.git
-$ cd hoist
-```
+* [Production](#production)
+* [Multinode Nodepool](#multinode-nodepool)
+* [Docker](#docker)
+* [Vagrant](#vagrant)
 
-To perform a full deploy:
+### Production
 
-```shell
-$ vagrant up
-```
+This is the live system providing the active BonnyCI service. A cron job runs the hoist `install-ci.yml` playbook every 15 minutes.
 
-To redeploy just the nodepool VM:
+### Multinode Nodepool
 
-```shell
-$ vagrant destroy nodepool
-$ vagrant up nodepool
-```
+Our nodepool configuration includes the multinode label `ubuntu-hoist` used for testing changes with the zuul `ubuntu-hoist` job. The zuul job deploys hoist on the single-use nodepool nodes and validates the result for every pull request. This environment closely mirrors production.
 
-To inspect the zuul VM:
+### Docker
 
-```shell
-$ vagrant ssh zuul
-$ # netstat, tcpdump, tail logs, etc.
-$ logout
-```
+We have a TravisCI test that deploys hoist in a docker container and validates the result. This uses the `allinone` inventory to deploy all components in a single container.
```

Comment:
The docker section of our lore should provide developers with enough information to create their own docker-base infrastructure. Please include this.